tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -4849,6 +4960,16 @@ int main(int argc, char *argv[]) {
                                       options::ConformingMethodListExpectedTypes);
     break;
 
+  case ActionType::SignatureHelp:
+    if (options::CodeCompletionToken.empty()) {
+      llvm::errs() << "signature help token name required\n";
+      return 1;
+    }
+    ExitCode = doSignatureHelp(
+        InitInvok, options::SourceFilename, options::SecondSourceFilename,
+        options::CodeCompletionToken, options::CodeCompletionDiagnostics);
```

Comment:
Should I add separate `-signature-help-token` and `-signature-help-diagnostics` flags to avoid confusion or is relying on the existing code completion options fine here? I feel neutral to be honest.